.github/actions/setup-ai/action.yml

```diff
@@ -0,0 +1,19 @@
+name: setup-ai-server
+description: "Setup AI server for testing"
+
+runs:
+  using: composite
+  steps:
+    - name: Install ollama
+      shell: bash
+      run: curl -fsSL https://ollama.com/install.sh | sh 
+    
+    - name: Start ollama
+      if: steps.cache-ollama.outputs.cache-hit != 'true'
+      shell: bash
+      run: ollama pull tinydolphin
+  
+    - name: Start litellm
+      shell: bash
+      run: poetry run litellm --config src/litellm/proxy_config_for_testing.yaml &
```

Comment:
How do we turn this off? Because if we do, this action will run forever.

Reply:
I'm thinking what you might need to do is to run llm separately in a container. See https://docs.github.com/en/actions/using-containerized-services/about-service-containers

Reply:
https://hub.docker.com/r/litellm/litellm

Reply:
For now, it looks like test is happy and GH action can turn this off. We can revisit in a separate ticket.